ipaserver/install/cainstance.py

```diff
@@ -282,7 +282,7 @@ class CAInstance(DogtagInstance):
     tracking_reqs = (('auditSigningCert cert-pki-ca', None),
                      ('ocspSigningCert cert-pki-ca', None),
                      ('subsystemCert cert-pki-ca', None),
-                     ('caSigningCert cert-pki-ca', 'ipaCACertRenewal'))
+                     ('caSigningCert cert-pki-ca', None))
```

Comment:
By removing the one special snowflake, you can remove the second part of each tuple (also in `KRAInstance`) and simplify `DogtagInstance.configure_renewal()` as well